build/common.props

```diff
@@ -1,9 +1,10 @@
 <Project>
   <PropertyGroup>
     <!-- Packages can have independent versions and only increment when released -->
-    <Version>3.0.29$(VersionSuffix)</Version>
+    <VersionSuffix>-dc</VersionSuffix>
+    <Version>3.0.30$(VersionSuffix)</Version>
     <ExtensionsStorageVersion>4.0.5$(VersionSuffix)</ExtensionsStorageVersion>
-    <HostStorageVersion>4.0.3$(VersionSuffix)</HostStorageVersion>
+    <HostStorageVersion>4.0.4$(VersionSuffix)</HostStorageVersion>
```

Comment:
Will revert the changes in this file before checking in - these are required to build test packages versions which I've published to the test feed to allow dependent projects (Functions Host/Track 2 extensions) to compile on CI.